src/test/resources/sql/scheduledPrisonerReturnFromCourt_init.sql

```diff
@@ -0,0 +1,139 @@
+INSERT INTO OFFENDERS (OFFENDER_ID, OFFENDER_NAME_SEQ, ID_SOURCE_CODE, LAST_NAME, NAME_TYPE, FIRST_NAME,
```

Comment:
You don't need to populate most of the columns in these tables.  Certainly not the audit columns and very few of the other columns.  Adding these values makes it harder to see what's important for the test(s).

Reply:
done